### ISSUE TITLE
TokenizerGpt3.Encode  incorrect

### ISSUE THREAD

**Issue by 3400442579** [betalgo/openai#122]
**Describe the bug**
TokenizerGpt3.Encode incorrect

**Your code piece**
```
 text = """
                Many words map to one token, but some don't: indivisible.

                Unicode characters like emojis may be split into many tokens containing the underlying bytes: 🤚🏾

                Sequences of characters commonly found next to each other may be grouped together: 1234567890
                """;
            int n=TokenizerGpt3.Encode(text).Count;
```
**Result**
= 260

**Screenshots**
![image](https://user-images.githubusercontent.com/34226979/222658938-5afbb9f6-941a-4b6e-85fd-3c332ebfa33c.png)

**Desktop (please complete the following information):**
 - OS: [Windows]
 - Language [c#]
 - Version [6.7.0]

**Additional context**

**Comment by kayhantolga** [comment]
That's interesting, I thought I solved this issue in 6.7.0 . Thanks for reporting, I will have a look

**Comment by kayhantolga** [comment]
solved in v6.7.2
